engine/src/RendererGLFW_GL46.cpp

```diff
@@ -0,0 +1,166 @@
+#include "Engine/Core/Rendering/Renderer/RendererGLFW_GL46.hpp"
+
+#include <glad/glad.h>
+#include <GLFW/glfw3.h>
+
+#include "Engine/Core/Debug/Log.hpp"
+#include "Engine/Core/Rendering/Window/WindowGLFW.hpp"
+#include "Engine/Core/Tools/Format.hpp"
+
+using namespace Engine::Core::Renderering;
+using namespace Engine::Core::Debug;
+using namespace Engine::Core::Tools;
+using namespace std;
+
+inline void APIENTRY GLDebugMessageCallback(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length,
```

Comment:
I've been looking for a way to simplify this function into something shorter and more efficient, but I saw that some of OpenGL's enum are not numbered contiguously. Maybe we can change them a bit in glad.h to be able to use some `const char* strings[]` in which we would keep the strings, and access them with strings[type] for instance?

Reply:
GLDebugMessage will be used rarely and does not constitute a bottleneck, so I think that we can keep this version